storage.ml

```diff
@@ -0,0 +1,71 @@
+type t = { version : int; users : User_model.user list }
```

Comment:
I love that this is versioned. :-)

Reply:
That's what I learned... once you write data to disk, better put a number next to it so that a future piece of code can revise / migrate / and don't have to abandon data written in the past ;)